packages/node_modules/@webex/widget-message/src/selector.js

```diff
@@ -1,10 +1,37 @@
 import {createSelector} from 'reselect';
+import {OrderedMap} from 'immutable';
 
 const getSparkInstance = (state, props) => props.sparkInstance || state.spark.get('spark');
 const getSparkState = (state) => state.spark.get('status');
 const getConversation = (state) => state.conversation;
 const getParticipants = (state) => state.conversation.get('participants');
-const getActivities = (state) => state.conversation.get('activities');
+const getConversationThreadActivities = (state) => state.conversation.get('threadActivities');
+const getConversationNonThreadActivities = (state) => state.conversation.get('sortNonThreadActivities');
+const getActivities = createSelector(
```

Comment:
Could we store these in the convo redux store this way via the reducer so we don't have to do the computation in every selector?